skimage/transform/ring_detector.py

```diff
@@ -0,0 +1,27 @@
+from ._ring_detector import RidgeHoughTransform
+
+__all__ = ['ring_detector']
```

Comment:
The all doesn't include `RidgeHoughTransform`, but the example uses it directly.  If it is supposed to be exposed, I think the `params` need to be hidden away/defined with the cdef.  You could make them `cdef public ...` to allow the user to modify the parameters (although I wonder a bit if the parameters shouldn't just be arguments to the `ridge_hough.rings_detection` call).  As to the `img_preprocess` call, it could probably be included into the `__cinit__`.